Troupon.Catalog.Api/DependencyInjectionExtensions/SwaggerHelper.cs

```diff
@@ -0,0 +1,135 @@
+using System;
```

Comment:
I think this will be a duplicate. All this have been moved into Okt.Infrastructure.

Did you add stuff in this or it's only a copy paste ?

Reply:
I think it was a copy paste, or maybe was something with the merge. I will delete it since we already have it in Okt.Infrastructure

Reply:
Good ! I just merger it in 3 projects, so we can make it work rapidly just after merge. Or even you can take "dev" and get it back in your branch before merge